core-blocks/heading/edit.native.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { RichText } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+
+const minHeight = 50;
+
+class HeadingEdit extends Component {
+	render() {
+		const {
+			attributes,
+			setAttributes,
+			style,
+			className,
+		} = this.props;
+
+		const {
+			align,
+			content,
+			level,
+			placeholder,
+		} = attributes;
+
+		const tagName = 'h' + level;
+
+		return (
+			<View>
+				<RichText
+					wrapperClassName="wp-block-heading"
+					tagName={ tagName }
+					content={ { contentTree: attributes.content, eventCount: attributes.eventCount } }
+					style={ {
+						minHeight: Math.max( minHeight, typeof attributes.aztecHeight === 'undefined' ? 0 : attributes.aztecHeight ),
```

Comment:
is there any way to move this `minHeight` setting to the `RichText`. I don't think that the block itself should contain `aztecHeight` attribute. It should be an internal implementation detail. It will cause all `Edit` block to fully re-render, where what we want is rather to re-render `RichText` when this height changes.

Reply:
I think that makes sense. Should we do it in a separate PR though?

Reply:
Sure thing, just wanted to raise it as I saw the same logic in `Paragraph` :)